gtfs-realtime/spec/en/reference.md

```diff
@@ -251,6 +251,27 @@ The degree of passenger occupancy for the vehicle.
 | _**FULL**_ | _The vehicle is considered full by most measures, but may still be allowing passengers to board._ |
 | _**NOT_ACCEPTING_PASSENGERS**_ | _The vehicle can not accept passengers._ |
 
+As an example, below is the conversion table from percentage of seats taken to OccupancyStatus published by the [Transport for New South Wales](http://www.transport.nsw.gov.au/) (TfNSW). TfNSW estimates the number of seat taken by using carriage weight and the percentage of seats taken can be more than 100%. Normalized percentage of seats taken is also provided in the third column.
+
+OccupancyStatus | Percentage (%) | Normalized Percentage (%)
+-- | -- | --
+MANY_SEATS_AVAILABLE | 0-70 | 0-53
+FEW_SEATS_AVAILABLE | 71-99 | 54-75
+STANDING_ROOM_ONLY | 100-114 | 76-87
+CRUSHED_STANDING_ROOM_ONLY | 115-130 | 88-99
+FULL | >131 | >100
+
+Here is the recommended conversion from observing other transit agencies:
+
+OccupancyStatus | Percentage (%)
```

Comment:
Let's remove this last table for now - we have a small sample size of producers and personally I'd prefer to avoid making recommendations